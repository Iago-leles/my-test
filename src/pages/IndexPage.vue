<template>
  <q-page class="q-pa-lg">
    <div class="row justify-center q-col-gutter-md">
      <div class="col-12 col-md-10">
        <hotel-search />
      </div>

      <hotel-order-filter class="col-12 col-md-10" />

      <hotel-list class="col-12 col-md-10" />
    </div>
  </q-page>
</template>

<script setup lang="ts">
import HotelSearch from 'components/HotelSearch.vue'
import HotelList from 'components/HotelList.vue'
import HotelOrderFilter from 'components/HotelOrderFilter.vue'
</script>
