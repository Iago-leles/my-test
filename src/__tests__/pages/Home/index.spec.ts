import { mount } from '@vue/test-utils'
import { describe, it, expect } from 'vitest'

import IndexPage from '@/pages/IndexPage.vue'

describe('IndexPage.vue', () => {
  it('should match snapshot', () => {
    const wrapper = mount(IndexPage)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
