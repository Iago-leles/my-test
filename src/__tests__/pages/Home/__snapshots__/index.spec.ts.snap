// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`IndexPage.vue > should match snapshot 1`] = `""`;
